addons/hr_recruitment_integration_monster/data/monster_industry_data.xml

```diff
@@ -0,0 +1,329 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="monster_industry_28" model="monster.industry">
```

Comment:
Remove this whole model and add a monster_id field on res.partner.industry model instead.

Reply:
removed